### ISSUE TITLE
Updates for VS2022.

### ISSUE THREAD

**Issue by niemyjski** [microsoft/VSSDK-Extensibility-Samples#237]
Are there any plans to update these samples for VS2022?

**Comment by DamianSuess** [comment]
yes, a lot of these samples are conflicting and do not match VS2022. It may be ideal to have subfolders to differentiate between VS2019 and VS2022 to showcase the best practices.

**Comment by EgorBo** [comment]
A year passed - still none of them build on vs2022 😞

**Comment by CoolIOT** [comment]
Still none of them build on Visual Studio Enterprise 2022 (64-bit) - Version 17.5.4 😞
